### PR TITLE
This is Blog section with all Crude operations

### PR DESCRIPTION
**Pull request by Abdulkeza**
# What does this PR do?
This PR Concerned with All crudes operations for a blog

# Description of Task to be completed?
- [x]  Create a post
- [x] View a list of posts
- [x] Retrieve one post 
- [x] Update post
- [x] Delete post  

# How should this be manually tested?
- [x] Postman
- [x] Thunder client 

  [CLONE](https://github.com/Abdulkeza/API-Backend.git) 

# Any background context you want to provide?
N/A

# What are the relevant pivotal tracker stories?
[jgQvNSNm](https://trello.com/b/jgQvNSNm/capston-project-first-and-second-modules)

# Questions:
N/A

### CODE REVIEW

File: controllers/BlogController.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    .then((result) => {
+      //   res.render("index", { title: "All Blogs", blogs: result });
+      res.json({ title: "All Blogs", blogs: result });
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+//DISPLAYING ONE POST
```
Comment (Jkadhuwa) [inline controllers/BlogController.js:19]: There is a lot of unnecessary comments. Put comments on complex code not every line of code. You can use jsdoc

File: controllers/BlogController.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+export const blog_create_post = async (req, res) => {
+  const blog = new Blog(req.body); //this is the insitance of Blog ---in blog.js
+
+  //check an empty submission or invalid
+  const { error } = blogValidation(req.body);
+  if (error) return res.status(400).send(error.details[0].message);
+
+  //save  post
+  try {
+    blog.save().then((result) => {
```
Comment (Jkadhuwa) [inline controllers/BlogController.js:48]: Use the await chaining the `.then`

**Review by Jkadhuwa — CHANGES_REQUESTED** [review]
Nice work so far. I've left some comments below.

-  Also make use of async-await it is much easier to read rather than chaining the '.then'

File: controllers/BlogController.js (hunk 2 above)
Comment (Abdulkeza) [reply controllers/BlogController.js:48]: yes

File: controllers/BlogController.js (hunk 1 above)
Comment (Abdulkeza) [reply controllers/BlogController.js:19]: noted

**Pull request merged**